test/PerfTest/WebApiPerformance.Service/WebApiPerformance.Service.csproj

```diff
@@ -22,22 +22,11 @@
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
     <TestType Condition=" '$(TestType)' == '' ">Private</TestType>
+    <Use64BitIISExpress />
   </PropertyGroup>
```

Comment:
Can remove?

Reply:
That‘s from VS automatically. I revert it